lib/airports/loader.ex

```diff
@@ -0,0 +1,63 @@
+NimbleCSV.define(AirportsParser, separator: ",", escape: "\"")
+
+defmodule Airports.Loader do
+  @moduledoc """
+  Load airports from a CSV file
+  """
+  alias Airports.Airport
+  # Tag as external_resource to ensure this module recompiles when this file changed
+  @external_resource Path.join([:code.priv_dir(:airports), "airports_with_tz.csv"])
+
+  def source_path, do: @external_resource
+
+  def run do
+    source_path()
+    |> File.stream!()
+    |> AirportsParser.parse_stream(skip_headers: true)
```

Comment:
Could you add a changelog entry for this addition as well?

Reply:
Sure thing